code/modules/mining/voucher_sets.dm

```diff
@@ -0,0 +1,72 @@
+/datum/voucher_set
```

Comment:
Also if you could auto-doc this actual datum that would be nice

Reply:
Autodoc added.